### DISCUSSION TITLE
Support table cell merging (ParsedownExtended)

### DISCUSSION THREAD

**Discussion by OlliL** [Zavy86/WikiDocs#220] [Ideas]
Consider cell merging for tables as described here:
- https://github.com/BenjaminHoegh/ParsedownExtended/issues/11
- https://github.com/erusev/parsedown/issues/682#issuecomment-466632520
- https://github.com/BenjaminHoegh/ParsedownExtended/blob/main/src/ParsedownExtended.php#L2209

The version bundled with WikiDocs is rather old (for whatever reason....) but it should already be supported in it

**Comment by leomoon** [comment]
This will require extensive testing as parsedown plugins work with specific versions.

We can test it against the basic-syntax doc.
[basic-syntax.md](https://github.com/user-attachments/files/17970534/basic-syntax.md)

**Reply by OlliL** [reply]
The old bundled version should already support it:
https://github.com/Zavy86/WikiDocs/blob/master/libraries/parsedown-extended-1.1.2-modified/ParsedownExtended.php#L1170

Nevertheless - there should be at least an automated blackbox test with a full-featured supported markdown document to execute the reffered "extensive testing", right? Updating versions shouldn't be something to be afraid of, right? :smiley:

**Reply by leomoon** [reply]
> The old bundled version should already support it: https://github.com/Zavy86/WikiDocs/blob/master/libraries/parsedown-extended-1.1.2-modified/ParsedownExtended.php#L1170
> 
> Nevertheless - there should be at least an automated blackbox test with a full-featured supported markdown document to execute the reffered "extensive testing", right? Updating versions shouldn't be something to be afraid of, right? 😃

Yea, I included the basic-syntax.md that has all wiki|docs features to test.

**Comment by leomoon** [comment]
I'll have an MR soon. Here's the test:
```
| Header 1 | Header 2 | >      | Header 3 |
|----------|----------|--------|----------|
| Cell A   | >        | Cell B | Cell B   |
| ^        | Cell C   | Cell D | Cell D   |
| Cell E   | Cell F   | Cell G | Cell G   |
| ^        | >        | Cell H | Cell H   |
```

and how it looks
![image](https://github.com/user-attachments/assets/af7bebf8-552c-482a-a516-c16d0a5c7659)

**Reply by OlliL** [reply]
Looks great imho! :+1:

**Reply by ffiesta** [reply]
> Looks great imho! 👍

👍

**Comment by leomoon** [comment]
this added.
